packages/arb-node-core/staker/staker.go

```diff
@@ -304,7 +305,8 @@ func (s *Staker) Act(ctx context.Context) (*arbtransaction.ArbTransaction, error
 
 	// Don't attempt to create a new stake if we're resolving a node,
 	// as that might affect the current required stake.
-	creatingNewStake := rawInfo == nil && s.builder.TransactionCount() == 0
+	wouldCreateStake := rawInfo == nil && s.builder.TransactionCount() == 0
+	creatingNewStake := wouldCreateStake && effectiveStrategy >= StakeLatestStrategy
```

Comment:
I think we also want to avoid underfunded defensive validators from failing here. Optimally they'd be able to function without a stake and then block waiting for funding if they don't have enough when they want to challenge

Reply:
This does that - `DefensiveStrategy < StakeLatestStrategy`, and their effective strategy would only reach `StakeLatestStrategy` and report an issue when a fault exists

Reply:
Ah of course